shared-test/src/main/java/uk/gov/di/authentication/sharedtest/helper/AuthAppStub.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.di.authentication.sharedtest.helper;
+
+import org.apache.commons.codec.binary.Base32;
+import uk.gov.di.authentication.shared.helpers.NowHelper;
+
+import javax.crypto.Mac;
+import javax.crypto.spec.SecretKeySpec;
+
+import java.util.concurrent.TimeUnit;
+
+public class AuthAppStub {
+    private final int CODE_DIGITS = 6;
+    private final long TIME_WINDOW_IN_MILLISECONDS = TimeUnit.SECONDS.toMillis(30);
+
+    // secret must parse as valid base32, or else an exception will be thrown by the library
```

Comment:
Do we need this comment? Could we catch any exception and then log a message out instead?